Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Recurring_rare_domain_access_from_an_unsigned_process.yml

```diff
@@ -0,0 +1,933 @@
+          "y": 9
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: 5268de6d-14fa-4b08-9eee-3d893efeef78
+    type: condition
+    task:
+      id: 5268de6d-14fa-4b08-9eee-3d893efeef78
+      version: -1
+      name: Check for Bad Reputation
+      description: Check if the process hash or domain has a suspicious/malicious reputation.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "20"
+      "yes":
+      - "21"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: greaterThanOrEqual
+          left:
+            value:
+              complex:
+                root: DBotScore
+                accessor: Score
+                transformers:
+                - operator: uniq
+            iscontext: true
+          right:
+            value:
+              simple: "2"
+        - operator: isNotEqualString
+          left:
+            value:
+              simple: WildFire.Verdicts.VerdictDescription
+            iscontext: true
+          right:
+            value:
+              simple: Benign
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 60,
+          "y": 133
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: 8dd4d58a-0f57-4c14-81d6-45221f4b5f1f
+    type: condition
+    task:
+      id: 8dd4d58a-0f57-4c14-81d6-45221f4b5f1f
+      version: -1
+      name: Check Investigation Verdict
+      description: "Checks if any 2 heuristics match the Malicious condition:\n\n- Risky Path\n- Risky Commandline\n- Low Prevalence\n- Related Incidents Found"
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "8"
+      Malicious:
+      - "7"
+    separatecontext: false
+    conditions:
+    - label: Malicious
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: .
+                transformers:
+                - operator: If-Elif
+                  args:
+                    conditions:
+                      value:
+                        simple: |-
+                          [
+                            {
+                              "condition": "#{CommandLineAnalysis.risk} != 'Low Risk' and  #{Core.AnalyticsPrevalence.Process.value} == [false]",
+                              "return": "True"
+                            },
+                            {
+                              "condition": "#{isRiskyPath} and  #{Core.AnalyticsPrevalence.Process.value} == [false]",
```

Comment:
I checked it, but I want to double-check: 'isRiskyPath' won't be created if no value is passed. correct?

Reply:
yes exactly